man/git-reauthor.md

```diff
@@ -0,0 +1,83 @@
+git-reauthor(1) -- Rewrite history to change author's identity
```

Comment:
I personally prefer my version:
- I don't want to put too many details in that short description. No need to mention a "committer id" there while many people don't know the existence of those 2 different type of identities. Same with mentioning there that it rewrites commits and tags.
- We better use the word rewrite I think, which is used within the Git documentation for filter-branch.

I've put those details you mention in the man page.